### ISSUE TITLE
HSlider: use pixels instead of percentages for touchArea default

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/sun#254]
In light of https://github.com/phetsims/sun/issues/251#issuecomment-242508522, HSlider's defaults for touchArea should be changed to pixels instead of percentages. This will certainly affect many clients.

**Comment by pixelzoom** [comment]
The bit of code to be changed is at line 162:

```js
 // Touch area for the default thumb. If a custom thumb is provided, the client is responsible for its touchArea.
    if ( !options.thumb ) {

      if ( options.thumbTouchAreaXDilation === null ) {
        options.thumbTouchAreaXDilation = 0.5 * thumb.width;
      }

      if ( options.thumbTouchAreaYDilation === null ) {
        options.thumbTouchAreaYDilation = 0.25 * thumb.height;
      }

      // touch area dilated along X and Y directions
      thumb.touchArea = thumb.bounds.dilatedXY( options.thumbTouchAreaXDilation, options.thumbTouchAreaYDilation );
    }
```

**Comment by pixelzoom** [comment]
Proposal: Change the default touch dilations to be 0.5 and 0.25 of the default thumb width and height respectively.  I.e.,

``js
     thumbSize: new Dimension2( 22, 45 ),
     ...
      thumbTouchAreaXDilation: 11,
      thumbTouchAreaYDilation: 11.25,
```

This will preserve the existing behavior for clients that use the default thumb size. And I can identify clients that might be affected by searching for "thumbSize:".

@samreid Do you forsee any problems with this plan?

**Comment by pixelzoom** [comment]
45 occurrence of "thumbSize:" in 25 sims.

**Comment by samreid** [comment]
Likewise I see 3 places where `thumbNode` is passed in to `HSlider`.  If those thumbs have different sizes, the behavior could be a bit different.  (Not a reason to invalidate your proposal, but they are cases that should probably be checked after the changes are made).

I don't see any other problems with the plan, but would recommend to test a case or two manually and make sure things are matching up as expected, perhaps by console.logging the touch area before/after to make sure it is preserved.

**Comment by pixelzoom** [comment]
@samreid said:
> Likewise I see 3 places where thumbNode is passed in to HSlider.

When a custom thumbNode is provided, the client is responsible for its pointer areas, as indicated in the documentation:

```js
55 // Dilation of touchArea for default thumb, ignored for custom thumb.
```

**Comment by samreid** [comment]
I see that now, thanks for showing me.  Everything seems good to proceed.

**Comment by pixelzoom** [comment]
I might as well add mouseArea options while I'm at it.

**Comment by pixelzoom** [comment]
Issues have been created for all sims that require adjustment, see list of issues immediately above. These are the sims that create one or more HSliders with the `thumbSize` option. They will also need to specify `thumbTouchAreaXDilation` and `thumbTouchAreaXDilation`.

**Comment by pixelzoom** [comment]
I took care of all the sims that didn't have a clear primary developer (color-vision, etc.)  The remaining issues are assigned to their respective primary developer(s).

**Comment by pixelzoom** [comment]
@jbphet selected at random to review.

**Comment by jbphet** [comment]
The commits all look good.  I tested by running the sun demo with ```showPointerAreas``` and changing the mouse and touch area dilation amounts in the HSlider options in ComponentsView.js, and everything worked correctly.  I also did some testing on States of Matter yesterday and noticed some asymmetry in the touch area, which @pixelzoom addressed in the commit shown just above.  Bottom line: Looks good, closing.  Thanks for doing this @pixelzoom.
